templates/bake/Plugin/src/Plugin.php.twig

```diff
@@ -76,4 +77,17 @@ class Plugin extends BasePlugin
 
         return $middlewareQueue;
     }
+
+    /**
+     * Add commands for the plugin.
+     *
+     * @param \Cake\Console\CommandCollection $commands The command collection to update.
+     * @return \Cake\Console\CommandCollection
+     */
+    public function console(CommandCollection $commands) : CommandCollection
+    {
+        // Add your commands here
```

Comment:
Should we have the parent call here so that plugins continue to autodiscover commands?

Reply:
Good idea; I added that as well.